source/getting-started/installation-raspberry-pi-all-in-one.markdown

```diff
@@ -0,0 +1,52 @@
+---
+layout: page
+title: "Raspberry Pi All-In-One Installer"
+date: 2016-05-12 01:39
+comments: true
+sharing: true
+footer: true
+---
+Easily deploy a complete Home-Assistant server, complete with Websocket MQTT and Z-Wave support out of the box using Fabric!
+
+**Usage:**
+*Start with a fresh Raspbian Jessie/Jessie-Lite or Debian 8 install*
```

Comment:
I guess that this is not a hard requirement. Wouldn't it work with an existing installation of a debian-based system, too?